Gemfile

```diff
@@ -7,4 +7,6 @@ group :jekyll_plugins do
   gem 'jekyll-gist'
   gem 'jekyll-paginate'
   gem 'jekyll-algolia'
+  gem 'kramdown-parser-gfm'
+  gem 'webrick'
```

Comment:
Are these addition gems required?

Reply:
I'm have a look but I'm pretty sure it had to do with running Jekyll on my local. Will get back to you on it today

Reply:
Ah okay I found it!

When I pulled the site to run it locally and make sure the images loaded correctly I got errors when trying to serve the site

```
Dependency Error: Yikes! It looks like you don't have kramdown-parser-gfm or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- kramdown-parser-gfm' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/! 
  Conversion error: Jekyll::Converters::Markdown encountered an error while converting '_posts/2019-01-02-january-2nd-2019.md':
                    kramdown-parser-gfm
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    kramdown-parser-gfm
```

So I added those 2 dependencies and the site generated and previewed just fine